code/modules/mob/living/carbon/xenomorph/xenomorph.dm

```diff
@@ -517,3 +524,48 @@ Returns TRUE when loc_weeds_type changes. Returns FALSE when it doesn’t change
 		height *= gravity * 0.5
 
 	AddComponent(/datum/component/jump, _jump_duration = duration, _jump_cooldown = cooldown, _stamina_cost = 0, _jump_height = height, _jump_sound = sound, _jump_flags = flags, _jumper_allow_pass_flags = jump_pass_flags)
+
+/mob/living/carbon/xenomorph/equip_to_slot(obj/item/item_to_equip, slot, bitslot)
```

Comment:
Jesus christ. How about you move the shared behaviour onto the `equip_to_slot` parent from the human subtype instead of this massive copypasta?

Reply:
done